docs/ops/state/large_state_tuning.md

```diff
@@ -177,12 +188,11 @@ Flink provides some predefined collections of option for RocksDB for different s
 We expect to accumulate more such profiles over time. Feel free to contribute such predefined option profiles when you
 found a set of options that work well and seem representative for certain workloads.
 
-**Important:** RocksDB is a native library, whose allocated memory not from the JVM, but directly from the process'
+<span class="label label-info">Note</span> RocksDB is a native library, whose allocated memory not from the JVM, but directly from the process'
```

Comment:
RocksDB is a native library that allocates memory directly from the process, and not from the JVM.